templates/dt_template/README.md

```diff
@@ -0,0 +1,58 @@
+# RHOS Deployed Topology %NEW_DT_NAME%
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | %TODAY%      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Compute nodes                                 | vm    | 3      |
+| Galera DB node, replicating                   | vm    | 3      |
+| Hardware security module                      | bm    | 1      |
+
+## Services, enabled features and configurations
+| Service                                     | configuration                   | Lock-in coverage?  |
+| ------------------------------------------- | ------------------------------- | ------------------ |
+| RabbitMQ                                    | default                         | Must have          |
+| OVN                                         | default                         | Must have          |
+| RabbitMQ                                    | default                         | Must have          |
+| Cinder                                      | Ceph/RGW backend                | optional           |
+| Manila                                      | Ceph/Ganesha+MDS Backend        | Must have          |
+| Glance                                      | NetApp, Fibre channel           | Must have          |
+| Keystone                                    | Special appliance [redacted]    | Interchangable     |
+| FIPS Mode                                   | default                         | Must have/standard |
+
+## Considerations
+
+1. Glance requirement dictates specific lab
+2. HSM access limited geographically
+
+## Testing tree
+
+| Test framework   | Stage to run | Special configuration | Test case to report |
+| ---------------- | ------------ | --------------------- | :-----------------: |
+| Tempest/compute  | stage7       | Use rhel image        | 11223344            |
+| Tempest/scenairo | stage7       | Use cirros image      | 22334455            |
+| Jordan           | stage7       | None                  | 44556677            |
+| Rally/Browbeat   | stage8       | Use CS9 image         | 55667788            |
+| Tobiko/Faults    | stage9       | Use cirros image      | 33445566            |
+
+## Stages
```

Comment:
I would make this optional, especially if the stages are going to be "standardized"

Reply:
I think it's OK to think in terms of stages for now. In the long run I don't think they'll be standardized. This can be rebased if we switch to kustomize as seen here.

https://github.com/openstack-k8s-operators/architecture/pull/39

Reply:
My point was about having non-numerical names as in your patch, that's a way to standardize them (once you have a stage called "control-plane" of course people won't duplicate that, but improve it)